### ISSUE TITLE
[Question] Is it possible not to use AH game in Labs Testing at all?

### ISSUE THREAD

**Issue by yehormasalitin** [LTTLabsOSS/markbench-tests#16]
### What happened?

Hello,

My question is mainly about the game selected, and not to the program itself.

I noticed a while back that atomic heart was one of the games selected for benchmarking, and I didn't know where to better forward my question, so I hope that this is the most appropriate one.

AH is a very controversial game due to where it is being developed and where the revenue from this game goes, example:
https://www.denofgeek.com/games/atomic-heart-controversy-explained-boycott-protests-ukraine/ 

For me and my Ukrainian friends who watch various LTT channels, it was always sad to see this game being used (and effectively indirectly promoted) for benchmarking.

I don't ask to have a political stance here or anything like that. But please consider that out of all the recent games that were released this one is probably the only one that is that controversial, and I understand that you didn't know this part (it was only briefly mentioned a couple of times on TechLinked). 
Replacing this game for benchmarking would be something huge for your Ukrainian audience.

I just wanted to give you a bit of context from one slice of your audience.

Thank you

**Comment by BartVerm779** [comment]
This has nothing to do with the program MarkBench and this is completly the wrong place to adress this. LTT has a forum were you can adress this issue if you want to.

Even adding the tag "Bug", which this is clearly not a bug, but your opinion.

**Comment by yehormasalitin** [comment]
@BartVerm779 
Thank you, I will try to raise it there then.
As I said, I didn't know what is the best place to question that, and when I saw the AH folder in the repo - I thought that it is the place to ask as it one of the benchmarks.

"bug" was added not by me, but rather by GitHub issue configuration for this repo. There was no way to change that. Currently all questions and bugs are marked as bugs, the only difference is the title. It was not my intention to mark it as such.


"but your opinion"
Not quite an opinion, I rather have a context that creators of the projects might not be aware of previously, I provided some facts and a suggestion, nothing more.
